css/main.css

```diff
@@ -177,12 +205,12 @@ nav .user-greet .user-greet-msg {
 }
 
 .social-media-links > span {
-  color: #e20665;
+  color: #e30162;
```

Comment:
You have added #e30162 as `--color-events`, so could we use that here?

Reply:
yes we can use that here.